riscv/insns/c_jalr.h

```diff
@@ -3,3 +3,9 @@ require(insn.rvc_rs1() != 0);
 reg_t tmp = npc;
 set_pc(RVC_RS1 & ~reg_t(1));
 WRITE_REG(X_RA, tmp);
+
+if (p->extension_enabled(EXT_ZICFILP)) {
+  if (ZICFILP_xLPE(STATE.v, STATE.prv)) {
+    STATE.elp = ZICFILP_IS_LP_EXPECTED(insn.rvc_rs1());
```

Comment:
The `if (p->extension_enabled(EXT_ZICFILP))` can be merged into `ZICFILP_xLPE`.

Reply:
Fixed.